tests/integration/test_vulnerability_detector/test_feeds/archlinux/test_extra_tags_archlinux_feed.py

```diff
@@ -70,15 +124,97 @@ def modify_feed(test_values, request):
+          VULNERABILITIES table of CVE DB.
+
+    input_description:
+        - Test cases are defined in the custom_archlinux_feed.json file. This file contains two different kinds of
+          vulnerabilities. This test gets their configuration of the wazuh_invalid_archlinux_feed.yaml file.
+
+    expected_output:
+        - 'Number of inserted vulnerabilities is not the expected.'
+        - r'Expected, .*, Got: .*'
+
+    tags:
+        - vulnerability
+        - vulnerability_detector
+        - system_inventory
+    '''
     vd.check_feed_imported_successfully(wazuh_log_monitor=wazuh_log_monitor, log_system_name=vd.ARCH_LOG,
                                         expected_vulnerabilities_number=vd.ARCH_NUM_CUSTOM_VULNERABILITIES)
 
 
 @pytest.mark.parametrize('test_values', vd.EXTRA_TEST_VALUES, ids=vd.EXTRA_TEST_IDS)
 def test_extra_tags_arch_linux_feed(test_values, clean_vuln_tables, get_configuration, configure_environment,
                                     modify_feed):
-    """Check if Vulnerability Detector behaves as expected while importing Arch Linux JSON feed with extra tags."""
+    '''
+    description: Check if Vulnerability Detector behaves as expected when importing the Arch Linux JSON feed with extra
+                 tags. To do this, a vulnerability event is launched and check if the number of vulnerabilities inserted
+                 in the VULNERABILITIES table of CVE DB is the expected and finally check if the process is still
+                 running.
+
+    wazuh_min_version: 4.3.0
+
+    parameters:
+        - test_values:
+            type: tuple's list
+            brief: List of tests to be performed.
+        - clean_vuln_tables:
+            type: fixture
+            brief: Clean vulnerabilities tables.
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - modify_feed:
+            type: fixture
+            brief: Modify the Amazon Linux JSON feed by setting a test tag value.
+
+    assertions:
+        - Verify that the number of vulnerabilities is equal to the expected number of vulnerabilities inserted in the
+          VULNERABILITIES table of CVE DB and if the process is running.
+
+    input_description:
+        - Test cases are defined in the test_values list. Each item of the list contains a tag and a value. Based on the
+          tag the feed is checked to be imported successful or not. This test gets their configuration of the
+          wazuh_invalid_archlinux_feed.yaml file.
+
+    expected_output:
+        - 'Number of inserted vulnerabilities is not the expected.'
+        - r'Expected, .*, Got: .*'
+        - r'ERROR: The .* feed couldn't be parsed from .* file'
+        - r'ERROR: CVE database could not be updated.'
+        - r'Could not find the log event: .*'
```

Comment:
Add the `check_feed_imported_successfully` callback
```suggestion
        - r'INFO: .* The update of the .* feed finished successfully.'
        - r'ERROR: The .* feed couldn't be parsed from .* file'
        - r'ERROR: CVE database could not be updated.'
```

Reply:
Done in this [link](https://github.com/wazuh/wazuh-qa/commit/92b4e92a92d158ca4a5355cb342855605b51e726)